### PR TITLE
feat: Playlist management

### PR DESCRIPTION
**Pull request by ravikumar0403**
### change log: 
---
#### featues added:
- Create playlist
- Add to playlist
- Remove from playlist
- delete playlist

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
- good folder structure
- good use of react portals
- design look consistent.

File: src/components/PlaylistModal.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        if (pl._id === updatedList._id) {
+          return updatedList;
+        }
+        return pl;
+      })
+    );
+    setLoaders((prev) => prev.filter((loader) => loader !== playlist._id));
+  };
+
+  console.log(loaders);
```
Comment (tushargoyal812) [inline src/components/PlaylistModal.js:43]: you can remove console

**Review by tushargoyal812 — COMMENTED** [review]
code is very clean and to the point.
liked your variable naming.
file structure is very good and clean.

File: src/components/PlaylistModal.js (hunk 1 above)
Comment (ravikumar0403) [reply src/components/PlaylistModal.js:43]: done

**Pull request merged**